src/main/java/de/doubleslash/usb_led_matrix/view/SettingsView.java

```diff
@@ -0,0 +1,92 @@
+package de.doubleslash.usb_led_matrix.view;
+
+import de.doubleslash.usb_led_matrix.model.AvailabilityStatus;
+import javafx.fxml.FXML;
+import javafx.scene.control.ColorPicker;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.paint.Color;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Properties;
+
+public class SettingsView {
+   private static final Logger LOG = LoggerFactory.getLogger(ConfigurationView.class);
+
+   @FXML
+   private ColorPicker awayColor;
+
+   @FXML
+   private ColorPicker availableColor;
+
+   @FXML
+   private ColorPicker beRightBack;
+
+   @FXML
+   private ColorPicker busyColor;
+
+   @FXML
+   private ColorPicker doNotDisturbColor;
+
+   String filePath = "settings.properties";
+
+   @FXML
+   public void initialize() throws IOException {
+      loadProperty();
```

Comment:
Should be executed when client is started and not when settings view is opened.

Reply:
Done.